api/client/options.proto

```diff
@@ -187,4 +187,6 @@ message CommandLineOptions {
   repeated string labels = 28;
   // TransportSocket configuration to use in every request
   envoy.config.core.v3.TransportSocket transport_socket = 27;
+  // Do not perform the simple warmup call.
+  google.protobuf.BoolValue no_simple_warmup = 32;
```

Comment:
On a similar topic - why wouldn't we make it an opt-in right away? What are the use cases to have this enabled by default?

Reply:
I think that is a good idea, fixed in 5940d5601eaa9924782c57be5a67895a7a1dfdd5 (with a callout in the changelist)